app/controllers/users_controller.rb

```diff
@@ -21,10 +29,48 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by id: params[:id]
+    return if @user
+    flash[:warning] = t :show
+    redirect_to root_path
+  end
+
+  def update
+    @user = User.find_by id: params[:id]
+    if @user&.update_attributes user_params
+      flash[:success] = t :update
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @user.destroy
+    flash[:success] = t :delete
+    redirect_to users_url
+  end
+
   private
 
   def user_params
     params.require(:user).permit :name, :email, :password,
       :password_confirmation
   end
+
+  def logged_in_user
+    return if logged_in?
+      flash[:danger] = t :log_in
+      redirect_to login_url
+  end 
+
+  def correct_user
```

Comment:
Move this method to the application controller, so that it can be reusable to others controller.